components/authz-service/server/v2/project.go

```diff
@@ -108,6 +110,10 @@ func (s *state) CreateProject(ctx context.Context,
 	if err != nil {
 		if err == storage_errors.ErrConflict {
 			return nil, status.Errorf(codes.AlreadyExists, "project with ID %q already exists", req.Id)
+		} else if err == storage_errors.ErrMaxProjectsAllowed {
+			return nil, status.Errorf(codes.FailedPrecondition,
+				"while IAM V2 beta is in progress, there is a max project count of %d: "+
```

Comment:
```suggestion
				"Max of %d projects allowed while IAM v2 Beta is in progress."
```

I think you can take that delete a project part out too, since deleting a project is going to orphan resources to start, im not sure we want to encourage that.